app/web_conn.go

```diff
@@ -140,6 +149,23 @@ func (wc *WebConn) Pump() {
 	close(wc.pumpFinished)
 }
 
+// BlockingPump is the Windows alternative of Pump.
+// It creates two goroutines - one for reading, another
+// for writing.
```

Comment:
So we are creating one goroutine for writing and using the current one for reading which essentially blocks until disconnection.

Reply:
Yeah, this is the current behavior. No change from that.